rest_api/openshift_v1.adoc

```diff
@@ -9278,7 +9278,7 @@ Patch is provided to give a concrete name and type to the Kubernetes PATCH reque
 |certificate|provides certificate contents|false|string|
 |key|provides key file contents|false|string|
 |caCertificate|provides the cert authority certificate contents|false|string|
-|destinationCACertificate|provides the contents of the ca certificate of the final destination.  When using re-encrypt termination this file should be provided in order to have routers use it for health checks on the secure connection|false|string|
+|destinationCACertificate|provides the contents of the ca certificate of the final destination.  When using re-encrypt termination this file should be provided in order to have routers use it for health checks on the secure connection|true|string|
```

Comment:
@pweil- @ramr Could one of you weigh in on my above question about whether it's more accurate for this value to show true or false? Or point us in the right direction. :bow: